src/modules/person/domain/required-step-up-level-not-met.error.ts

```diff
@@ -0,0 +1,7 @@
+import { PersonDomainError } from './person-domain.error.js';
+
+export class RequiredStepUpLevelNotMetError extends PersonDomainError {
+    public constructor(entityId: string | undefined, details?: unknown[] | Record<string, undefined>) {
+        super(`The action could not be perfomed, bequase the required step up level was not met`, entityId, details);
```

Comment:
I don't think the parameter for `entityId` is used anywhere. Do we even need it? Also, should this be a PersonDomainError? Feels a lot more like an `AuthenticationDomainError`